darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -78,6 +78,7 @@
 CHECKPOINTS_FOLDER = "checkpoints"
 RUNS_FOLDER = "runs"
 INIT_MODEL_NAME = "_model.pth.tar"
+TFM_ATTRS_NO_PICKLE = {"model": None, "trainer": None}
```

Comment:
Nice! Maybe 1-2 comments here to explain that pickling a TFM will not pickle these attributes would be helpful.